miniapp/USERGUIDE.md

```diff
@@ -290,8 +290,7 @@ override fun onBackPressed() {
 
 The mini app is loaded with the specific custom scheme and custom domain in mini app view.
 
-In default, the external link is also loaded in mini app view.
-It is possible for hostapp to load this external link with its own webview / browser.
```

Comment:
I am not sure about the better place to write but it's good to mention like following:

"HostApp also can implement their own way by passing `MiniAppNavigator` object using `MiniApp.create(appId: String, miniAppMessageBridge: MiniAppMessageBridge, miniAppNavigator: MiniAppNavigator)` ". There is a sample implementation of `MiniAppNavigator` in [here](https://github.com/rakutentech/android-miniapp/blob/master/testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/WebViewActivity.kt)"

Reply:
let me update it.